templates/web/shropshire/footer_extra_js.html

```diff
@@ -0,0 +1 @@
+[% PROCESS 'footer_extra_js_base.html' cobrand_js=1 validation=1 %]
```

Comment:
You need highways=1 if you want the Highways England handling code.

Reply:
Locally, this is now showing the option of is this a highway, but not showing the categories when 'On the ...' selected. Not sure if that's my setup or I need something else.